src/intl-head.tsx

```diff
@@ -0,0 +1,28 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React, { ReactElement, ReactNode } from 'react';
+import { useAlternateLinks } from './hooks/useAlternateLinks';
+import { useCanonicalUrl } from './hooks/useCanonicalUrl';
+
+interface IntlHeadProps {
+  title: string;
```

Comment:
Why is `title` handled differently from all the other content that might go into `<head>`?

Reply:
You're right :) 
I actually don't need to pass it as a prop, it can be passed in children. Thanks